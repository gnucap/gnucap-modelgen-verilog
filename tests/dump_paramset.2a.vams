
`include "sub/disciplines.few"
`pragma modelgen expand-paramset

module resistor(a,b)
	electrical a, b;

	parameter real r = 1.;
	localparam real l = r;

	analog begin
		I(a,b) <+ V(a,b)/l;
	end
endmodule

paramset my_res resistor;
	parameter real r = 1.;
	.r = r;
endparamset;
