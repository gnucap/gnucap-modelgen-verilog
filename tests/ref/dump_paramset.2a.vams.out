// paramsets
module my_res(a, b);
  electrical a, b;
  parameter real r =  1.;
  localparam real __r = r;
  localparam real __l = __r;

  analog begin
    I(a, b) <+ (V(a, b) / __l);
  end

endmodule

